src/id3/v2/items/popularimeter.rs

```diff
@@ -46,6 +47,29 @@ impl Popularimeter {
 
 		content
 	}
+
+	/// Convert ID3v2 POPM frame bytes into a [`Popularimeter`].
+	pub fn from_bytes(bytes: &[u8]) -> Result<Self, LoftyError> {
```

Comment:
You can just copy the logic from here (without wrapping it in `FrameValue`):
https://github.com/Serial-ATA/lofty-rs/blob/f25ed9b6c7c920263c57ab327d023e517ae0e16a/src/id3/v2/frame/content.rs#L165-L186

And then change this:
https://github.com/Serial-ATA/lofty-rs/blob/f25ed9b6c7c920263c57ab327d023e517ae0e16a/src/id3/v2/frame/content.rs#L37

to:
```rust
"POPM" => Some(FrameValue::Popularimeter(Popularimeter::from_bytes(content)?)),
```

Be sure to update your branch, I just edited `parse_popularimeter` :slightly_smiling_face:.